### ISSUE TITLE
Another Potential DB error :warning:  :warning: Can not remove miners from the list(DB)

### ISSUE THREAD

**Issue by DerXanRam** [UnamSanctam/UnamWebPanel#346]
Hello unam. When there is [still DB error](https://github.com/UnamSanctam/UnamWebPanel/issues/344) in the webpannel, another problem is appearing. The web pannel can't remove miner and it prints this error

![error](https://github.com/UnamSanctam/UnamWebPanel/assets/101794261/a4e82aef-8693-4ba4-9ea8-71ec800fe9db)

I use the latest version of the webpannel.

**Comment by UnamSanctam** [comment]
Yes if the database is corrupt then naturally you shouldn't be able to delete anything, though if it's corrupted then you shouldn't really be able to see any miners in the first place.

**Comment by DerXanRam** [comment]
But other functionalities are working fine. And this problem are appearing after the new update. 

Please help

**Comment by UnamSanctam** [comment]
You said that your database was corrupted (malformed) in the other issue, so that is the problem.

**Comment by DerXanRam** [comment]
> You said that your database was corrupted (malformed) in the other issue, so that is the problem.

Hello bro. Sorry again for my late reply. This issue is opened because the database will corrupt when my miner number passes exactly 200. The other issue is, the web panel can't remove miner from the database.
